include/utils/ignore_warnings.h

```diff
@@ -33,6 +33,7 @@
 #pragma clang diagnostic ignored "-Wunused-private-field"
 #pragma clang diagnostic ignored "-Wextra"
 #pragma clang diagnostic ignored "-Wredundant-decls"
+#pragma clang diagnostic ignored "-Wcast-align"
```

Comment:
Did you happen to check whether `-Wcast-align` is new and hence should only be used with newer versions of clang?

Reply:
Yeah, it's as old as I could find docs for.

Reply:
Not sure whether it's only triggering for me now because the new clang catches more instances or because of the new OpenMPI just started using it.